### DISCUSSION TITLE
Cannot edit or create anything anymore

### DISCUSSION THREAD

**Discussion by Joly0** [ZoeyVid/NPMplus#453] [Q&A] [answered]
Hey, i have configured a proxy host for npm in npm and applied these settings:
![grafik](https://github.com/ZoeyVid/NPMplus/assets/13993216/bd2d2d34-f5d9-41eb-bcd6-95a1a92d9ae7)
But no i cannot edit or create anything anymore. I get for example this error message:
```2023/10/30 22:42:03 [error] 858#858: *3 [client 172.71.246.160] ModSecurity: Access denied with code 403 (phase 2). Matched "Operator `Ge' with parameter `5' against variable `TX:BLOCKING_INBOUND_ANOMALY_SCORE' (Value: `20' ) [file "/usr/local/nginx/conf/conf.d/include/coreruleset/rules/REQUEST-949-BLOCKING-EVALUATION.conf"] [line "176"] [id "949110"] [rev ""] [msg "Inbound Anomaly Score Exceeded (Total Score: 20)"] [data ""] [severity "0"] [ver "OWASP_CRS/4.0.0-rc2"] [maturity "0"] [accuracy "0"] [tag "anomaly-evaluation"] [hostname "xxx.xxx.xxx.xxx"] [uri "/api/nginx/proxy-hosts/1"] [unique_id "169870212339.129525"] [ref ""], client: 172.71.246.160, server: npm.test.com, request: "PUT /api/nginx/proxy-hosts/1 HTTP/2.0", host: "npm.test.com", referrer: "https://npm.test.com/nginx/proxy"```

**Comment by Zoey2936** [comment, marked as answer]
you either
1. need to disable CoreRuleSet using the unproxied UI on Port 81 (if you haven't changed it)
or
2. you open /data/etc/modsecurity/crs-setup.conf with nano, search for `tx.allowed_methods=`
uncomment everything needed for it and change it to and then restart npmplus:
```
SecAction \
    "id:900200,\
    phase:1,\
    pass,\
    t:none,\
    nolog,\
    setvar:'tx.allowed_methods=GET HEAD POST OPTIONS PUT PATCH DELETE'"
```

**Reply by Joly0** [reply]
Thx, your first suggestion worked
